src/calendar/header/style.ts

```diff
@@ -1,12 +1,24 @@
-import {StyleSheet, Platform} from 'react-native';
+import {StyleSheet, Platform, TextStyle, ViewStyle, ImageStyle} from 'react-native';
 import * as defaultStyle from '../../style';
 import {Theme} from '../../types';
 import constants from '../../commons/constants';
 
+type Styles = {
+  header: ViewStyle;
+  headerContainer: ViewStyle;
+  monthText: TextStyle;
+  arrow: ViewStyle;
+  arrowImage: ImageStyle;
+  disabledArrowImage: ImageStyle;
+  week: ViewStyle;
+  dayHeader: ViewStyle;
+  disabledDayHeader: ViewStyle;
+};
+
 export default function (theme: Theme = {}) {
   const appStyle = {...defaultStyle, ...theme};
   const flipStyle = constants.isRTL ? {transform: [{scaleX: -1}]} : undefined;
-  return StyleSheet.create({
+  return StyleSheet.create<Styles>({
```

Comment:
remove the `<Styles>` please.

Reply:
With the type, we get autocomplete in style fields based on type (`TextStyle`, `ViewStyle` etc).

For now, I followed your advice and removed the type `Styles` but maybe we should consider adding the `Styles` type in a future PR. What do you think @Inbal-Tish?